### ISSUE TITLE
Generator fails with "No Visual Studio instances found"

### ISSUE THREAD

**Issue by dealdiane** [T4MVC/R4MVC#118]
Project is targeting netcoreapp2.2 and sdk 2.2.105. Tried both the cli and nuget generators but both give out the same error message:

"No Visual Studio instances found. The code generation might fail"

The project used to target netstandard1.6 and was referencing an older version R4MVC which worked until the upgrade.

I'm on VS2019 15.9.7 if that makes a difference.

Any ideas?

**Comment by artiomchi** [comment]
Hi @dealdiane 

That's a bit strange. Could you try running this command in the cli, and let me know what's the output?
```
r4mvc vsinstances
```

Also, I just pushed a new build to NuGet that has an updated VS Locator (to try get it to support the new VS 2019)

**Comment by dealdiane** [comment]
Same thing with the latest version.

Here's the output:
```
R4Mvc Generator Tool v1.0.0-alpha2-00339

No Visual Studio / MSBuild instances found.
```

**Comment by dealdiane** [comment]
I figured this has nothing to do with `R4MVC` but rather with `Microsoft.Build.Locator`. 

I remember pausing a recent VS update - and eventually cancelled it. VS installer says I have that version installed which isn't true so I figured that could be the issue (corrupt install state?). Updating to the latest version to fix the corrupt install state fixed the issue.

For reference, this issue pointed me to the right direction: https://github.com/Microsoft/MSBuildLocator/issues/17
